neuraxle_tensorflow/tensorflow_v1.py

```diff
@@ -0,0 +1,109 @@
+"""
+Neuraxle Tensorflow V1 Utility classes
+====================================
+Neuraxle utility classes for tensorflow v1.
+..
+    Copyright 2019, Neuraxio Inc.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+"""
+import os
+from abc import abstractmethod
+
+import tensorflow as tf
+
+from neuraxle.base import BaseSaver
+
+
+class TensorflowV1ModelWrapperMixin:
```

Comment:
Should we instead, add a ModelWrapperMixin in neuraxle ? 

I have this in mind : 

```
class ModelWrapperMixin: 
   def strip():
         raise NotImplementedError()

   def get_model_info() -> Dict:
         raise NotImplementedError()
```

Reply:
@guillaume-chevalier ?

Reply:
`return { 'graph': self.graph, 'session': self.session }` for tensorflow

Reply:
I like the name `TensorflowV1ModelWrapperMixin`.